trin-types/src/enr.rs

```diff
@@ -79,3 +81,61 @@ impl ssz::Encode for SszEnr {
         self.rlp_bytes().to_vec().ssz_bytes_len()
     }
 }
+
+#[allow(clippy::unwrap_used)]
+pub fn generate_random_remote_enr() -> (CombinedKey, Enr) {
+    let key = CombinedKey::generate_secp256k1();
+
+    let mut rng = rand::thread_rng();
+    let ip = Ipv4Addr::from(rng.gen::<u32>());
+
+    let enr = EnrBuilder::new("v4")
+        .ip(ip.into())
+        .udp4(8000)
+        .build(&key)
+        .unwrap();
```

Comment:
I can't really think of a time when it's better to use `clippy::unwrap_used` than to switch to `expect()` with an explanation of why it won't ever panic in reasonable scenarios of runtime. When does building the ENR fail? Why is that impossible now?